### PR TITLE
Task 1: created unit and e2e tests for listing all movies

### PR DESCRIPTION
**Pull request by kfinberg7581**

### CODE REVIEW

**Comment by H-M-Sammons** [comment]
looks good

File: tests/unit/test_get_all_movies.py [hunk 1]
```diff
@@ -1 +1,31 @@
...
+#test what happens if the list displays content that wasn't entered in the create_movie()
+def test_get_all_movies_not_in_List():
+    #test the app, do not create a movie here
+    testing_app = app.test_client()
+
+    movie_repository.create_movie('The Last Jedi', 'Rian Johnson', 3)
+    response = testing_app.get('/movies')
+    #check that the title is different 
+    assert not b'<td>The Last Sith</td>' in response.data
+    assert b'<td>Rian Johnson</td>' in response.data
```
Comment (sumiv) [inline tests/unit/test_get_all_movies.py]: add assert not for all the other values

**Review by sumiv — CHANGES_REQUESTED** [review]
assert not for other values

**Review by sumiv — APPROVED** [review]
lgtm!

**Pull request merged**